gptel-context.el

```diff
@@ -84,6 +84,23 @@ context chunk.  This is accessible as, for example:
   :group 'gptel
   :type 'function)
 
+(defun gptel-context-add-current-kill (&optional arg)
+  "Add current-kill to gptel, accumulating if arg is non-nil"
+  (interactive "P")
+  (let ((kill (current-kill 0)))
+    (with-current-buffer (get-buffer-create "*current-kill*")
+      (if (not arg)
+          (kill-region (point-min) (point-max))
+        (goto-char (point-max))
+        (unless (bobp)
+          (unless (bolp)
```

Comment:
I don't understand the double `(unless (bolp) ...)`, but I'm assuming it's required?